AUTHORS

```diff
@@ -81,6 +80,8 @@ Dave Goodell, Cisco
   dgoodell@cisco.com
 David Daniel, Los Alamos National Laboratory
   ddd@lanl.gov
+David Solt, IBM
+  dsolt@us.ibm.com
```

Comment:
I reverted it anyway, so dave's checkin now adds a line, and my checkin removes it so the two commits together should be a no-op on AUTHORS